nautobot/ipam/tests/test_api.py

```diff
@@ -443,6 +443,25 @@ def test_list_available_prefixes(self):
         for i, p in enumerate(response.data):
             self.assertEqual(p["prefix"], str(available_prefixes[i]))
 
+    def test_prefix_display_value(self):
+        """
+        Test that the `display` field is correctly populated.
+        """
+        prefix = (
+            Prefix.objects.filter(ip_version=6)
+            .filter(prefix_length__lt=128)
+            .exclude(type=choices.PrefixTypeChoices.TYPE_CONTAINER)
+            .first()
+        )
+        if prefix is None:
+            self.fail("Suitable prefix fixture not found")
+        url = reverse("ipam-api:prefix-list")
+        self.add_permissions("ipam.view_prefix")
+
+        response = self.client.get(url, **self.header, QUERY_STRING="depth=1")
```

Comment:
I'm not familiar with the use of `QUERY_STRING` as a header parameter, does it actually do something here? I'd have expected a more usual pattern to be `url = f"{url}?depth=1"` instead.

Reply:
Hi Glenn, this was used due to the test client that's being employed (Django's `test.client.RequestFactory`), which traces back to [here](https://github.com/django/django/blob/main/django/test/client.py#L399).  If you'd prefer that I use the `url` string instead, I can refactor.

Reply:
Changed in [7cd2afd](https://github.com/nautobot/nautobot/pull/6190/commits/7cd2afd808a1ff2d5fdeb6a728954a31560a513e)